multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/AbstractKeycloakFilter.java

```diff
@@ -23,8 +28,19 @@
  */
 public abstract class AbstractKeycloakFilter implements Filter {
 
+  private static final String MACHINE_TOKEN_KIND = "machine_token";
```

Comment:
Is it possible to use `org.eclipse.che.multiuser.machine.authentication.shared.Constants.MACHINE_TOKEN_KIND` value here?